src/distances/u32.rs

```diff
@@ -0,0 +1,39 @@
+#[allow(clippy::type_complexity)]
+pub const METRICS: [(&str, fn(&[u32], &[u32]) -> f32); 2] = [
```

Comment:
This exists to make the `Dataset` tests work without chance of floating point error. This could be removed/modified without issue.